packages/bundle-source/package.json

```diff
@@ -28,7 +28,7 @@
     "@agoric/transform-eventual-send": "^1.4.1",
     "@babel/generator": "^7.6.4",
     "@babel/traverse": "^7.8.3",
-    "@rollup/plugin-commonjs": "^11.0.2",
```

Comment:
what about the later version conflicts with Windows usage?

Reply:
I run into https://github.com/rollup/plugins/issues/304

It may be the specific version of Electron that does this, but I think it's easier to back down for now.  It should be fixed in `@rollup/plugin-commonjs@^17.0.0` but I don't want to upgrade all the rollup things quite yet.

Reply:
gotcha.

That's definitely worth mentioning in the commit message.

Reply:
Okay, I've done that.

Reply:
well, maybe not worth turning off automerge...